README.md

```diff
@@ -23,7 +23,7 @@ const todoHandler = {
     ADD: (state: Todo[], newTodo: Todo) => [...state, newItem],
     TOGGLE: (state: Todo[], index: number) => [
         ...state.slice(0, index),
-        { state[index], completed: !state[index].completed },
+        { task: state[index], completed: !state[index].completed },
```

Comment:
I think this should be either `...state[index]` or `task: state[index].task`?

Reply:
You're right. I was not paying enough attention there. I updated the PR.